src/coreclr/tools/Common/TypeSystem/IL/ILImporter.cs

```diff
@@ -325,6 +327,8 @@ private void ImportBasicBlock(BasicBlock basicBlock)
 
                 ILOpcode opCode = (ILOpcode)ReadILByte();
 
+                StartImportingInstruction(opCode);
```

Comment:
This will mis-report opcodes with prefix. Is that reasonable?

Reply:
I think it would be better to move handling of the prefixed instruction from the switch statement in front of this call.

Reply:
Yeah, that would be a pretty odd API, that wasn't intentional. Updated to read and report the prefix.